lib/server.js

```diff
@@ -1303,19 +1303,24 @@ Server.prototype._routeErrorResponse = function _routeErrorResponse(
         return;
     }
 
-    self._emitErrorEvents(req, res, null, err, function emitError(emitErr) {
+    self._emitErrorEvents(req, res, null, err, function emitError() {
```

Comment:
What happened to `emitError`?

Reply:
I looked at `_emitErrorEvents`, and `emitErr is actually the same object as `err`,  just passed back. It seemed confusing as heck, so I removed it outright. In retrospect, I should probably remove calling this callback w/ err to begin with.